tests/unit_tests/test_league_rater.py

```diff
@@ -0,0 +1,282 @@
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN
+
+
+def test_new_score_victory_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (1800.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN + config.POSITIVE_BOOST
+
+
+def test_new_score_defeat_no_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (180.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN
+
+
+def test_new_score_defeat_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (60.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN - config.NEGATIVE_BOOST
+
+
+def test_new_score_victory_highest_division_no_boost(example_league):
+    current_score = LeagueScore(division_id=3, score=5, game_count=30)
+    player_rating = (240.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN
+
+
+def test_new_score_victory_highest_division_boost(example_league):
+    current_score = LeagueScore(division_id=3, score=5, game_count=30)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN + config.HIGHEST_DIVISION_BOOST
+
+
+def test_placement(example_league, unplaced_player_score):
+    rating = 150 - config.RATING_MODIFIER_FOR_PLACEMENT
```

Comment:
Maybe explain in a comment why you are subtracting this (so that the test is independent of the value of `RATING_MODIFIER_FOR_PLACEMENT`)?

One could also make an argument that the test shouldn't make assumptions on the internals of `_do_placement` and you should just pass in `rating = 150`, but I guess this way it's clearer why you arrive at nice scores like `5` and `10`.